examples/twittermap/web/app/websocket/TwitterMapServerToCloudBerrySocket.scala

```diff
@@ -8,7 +8,7 @@ import org.eclipse.jetty.websocket.api.annotations._
 import play.api.Logger
 import play.api.libs.json.{JsValue, Json}
 
-@WebSocket
+@WebSocket(maxTextMessageSize = 5 * 1024 * 1024)
```

Comment:
Is it possible we get this value from a .conf file?

Reply:
Yes.  But I don't think this change should be done in this PR, because it should be another bug of the TwitterMap Server.

Reply:
OK. I can explore it after you push the PR.

Reply:
@baiqiushi if you want to address this in a separate PR, you should put a `TODO` comment here.